rest-api/src/main/java/gov/cms/qpp/conversion/api/logging/AttachmentHashPartConverter.java

```diff
@@ -29,7 +29,7 @@ public String convert(ILoggingEvent event) {
 
 		try {
 			hashPart = getHashPart();
-		} catch (IOException | ServletException e) {
+		} catch (Exception e) {
```

Comment:
Were there other exceptions that need to be caught and logged here?  I'd rather not catch `Exception` and be more specific.

Reply:
agreed

Reply:
Good catch.  Only thing is that we cannot log here (even if we do it at a `INFO` or higher level).  Logback is smart enough to know that this piece of code is executing while constructing a log statement and therefore squelches any logging.  We also don't want to manually log anything here because it will become noisy, ruin the JSON format of our logs, and be hard to follow.